parson.c

```diff
@@ -1095,14 +1095,27 @@ static JSON_Value * parse_null_value(const char **string) {
 }
 
 /* Serialization */
-#define APPEND_STRING(str) do { written = append_string(buf, (str));\
-                                if (written < 0) { return -1; }\
-                                if (buf != NULL) { buf += written; }\
+#define __APPEND_STRING(buf, string, written) do { (written) = SIZEOF_TOKEN(string);\
```

Comment:
Why not put this logic in APPEND_STRING macro? Also, I would be cautious about using a __ prefix.

Reply:
ack; will adjust;

no idea why i did it like that;

Reply:
comment implemented;